src/app/components/contact-button/contact-dialog.component.scss

```diff
@@ -0,0 +1,106 @@
+@import "./src/resources/styles/_colors";
+
+md-dialog {
+    height: 70%;
+    width: 49%;
+    border-width: 2px;
+    border-color: skyblue;
+    background-color: rgba(80, 80, 80, 0.753);
+    border-width: 2px;
+    border-style: solid;
```

Comment:
the border width and color are repeated, and you can do it like this: `boder: 2px solid skyblue`